### PR TITLE
Documentation: cleanup and fix some typo

### PR DESCRIPTION
**Pull request by huikang**
Signed-off-by: Hui Kang <kangh@us.ibm.com>

# Contributing guidelines

Please read our [contribution workflow][contributing] before submitting a pull request.

[contributing]: https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#contribution-flow

### CODE REVIEW

File: Documentation/op-guide/runtime-configuration.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Before making any change, a simple majority (quorum) of etcd members must be ava
 All changes to the cluster must be done sequentially:
 
 * To update a single member peerURLs, issue an update operation
-* To replace a healthy single member, add a new member then remove the old member
+* To replace a healthy single member, remove the old member then add a new member
```
Comment (xiang90) [inline Documentation/op-guide/runtime-configuration.md]: why change this?

Comment (huikang) [inline Documentation/op-guide/runtime-configuration.md]: I read this from [1]. if I understand correctly, the old member should first be removed. Is my understanding correct?

[1] https://github.com/coreos/etcd/blob/b7cf080e2c77c5ca8f52c4d8e3fdd7c8ef1bfb5a/Documentation/faq.md#should-i-add-a-member-before-removing-an-unhealthy-member

Comment (xiang90) [inline Documentation/op-guide/runtime-configuration.md]: ah. ok.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**